src/Microsoft.Identity.Abstractions/DownstreamApi/IAuthorizationHeaderProviderExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Abstractions
+{
+    /// <summary>
+    /// Provides a method to create an authorization header for calling protected web APIs 
+    /// on behalf of a user or the application itself.
+    /// </summary>
+    public interface IAuthorizationHeaderProviderExtension
```

Comment:
Generally, when following this pattern, this interface would also implement the base interface.